syntax = "proto3";

package suggestions;

service BooksDatabase {
    rpc Read (ReadRequest) returns (ReadResponse) {}
    rpc Write (WriteRequest) returns (WriteResponse) {}
}

message ReadRequest {
    string key = 1;
}

message ReadResponse {
    int32 value = 1;
}

message WriteRequest {
    string key = 1;
    int32 value = 2;
}

message WriteResponse {
    bool success = 1;
}


